rasa_core/server.py

```diff
@@ -508,8 +507,24 @@ def evaluate_stories():
                          "Evaluation could not be created. Error: {}"
                          "".format(e))
 
-    @app.route("/domain",
-               methods=['GET', 'OPTIONS'])
+    @app.route("/jobs", methods=['POST', 'OPTIONS'])
+    @requires_auth(app, auth_token)
+    @cross_origin(origins=cors_origins)
+    def train_stack():
+        from rasa.cli.train import train
+        from argparse import Namespace
+
+        """Train a Rasa Stack model."""
```

Comment:
should be below the function `def`

Reply:
hmm, thought it was a convention because it's the same in `evaluate_stories()` above. I'll change both